_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -173,13 +244,13 @@ Programatically associated `p` element that is neither visible nor included in t
 The `label` element is associated with an HTML element that does not have a form field semantic role.
 
 ```html
-<label for="fname" style:"display:none;">First name:</label>
+<label for="fname">First name:</label>
 <p id="fname"/>
 ```
 
 #### Inapplicable example 4
```

Comment:
Add Inapplicable test case where a `p` element acts as visible label but is not programatically determinable.